### DISCUSSION TITLE
kygisserver requests

### DISCUSSION THREAD

**Discussion by ianhorn** [sindresorhus/got#2252] [General]
I recently ran some stats on one of our servers (kygisserver.ky.gov/arcgis) and I noticed that `got (https://github.com/sindresorhus/got)` was 7th on our browser list.  I have never noticed before and was just curious of how you used got and find out what else it can do.  

Our data is free and open to the public for consumption.  The browser just caught my interest.

**Comment by sindresorhus** [comment]
Got isn't a browser - it's a popular Node.js HTTP client library used by developers to make programatic requests to APIs and servers. It's similar to `curl` or `wget`, but for Node.js apps.

What you're seeing in your logs is likely developers or automated systems using Got to interact with your ArcGIS server's API.
